api/identifiers/views.py

```diff
@@ -0,0 +1,98 @@
+from modularodm import Q
+from rest_framework import generics, permissions as drf_permissions
+
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from api.base import permissions as base_permissions
+from api.base.views import JSONAPIBaseView
+from api.base.filters import ODMFilterMixin
+
+from api.registrations.views import RegistrationMixin
+from api.identifiers.serializers import IdentifierSerializer
+
+from api.nodes.permissions import (
+    IsPublic,
+    ExcludeWithdrawals,
+)
+
+from website.identifiers.model import Identifier
+
+
+class IdentifierList(JSONAPIBaseView, generics.ListAPIView, RegistrationMixin, ODMFilterMixin):
+    """List of identifiers for a specified node. *Read-only*.
+
+
+   ##Identifier Attributes
+
+    OSF License entities have the "licenses" `type`.
+
+        name           type                   description
+        ----------------------------------------------------------------------------
+        category       string                 e.g. 'ark', 'doi'
+        referent       link                   object to which the identifier points
+        value          string                 the identifier value itself
+
+
+    ##Links
+
+    See the [JSON-API spec regarding pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+
+    ##Actions
+
+    *None*.
+
+    ##Query Params
+
+     Identifiers may be filtered by their category.
+
+    #This Request/Response
+
+    """
+
+    permission_classes = (
+        IsPublic,
```

Comment:
Is this accurate? I don't think we use this much, so I don't have a good feel for when we would use it. Is this saying that we can only read identifiers for public resources?